west.yml

```diff
@@ -57,6 +59,9 @@ manifest:
       path: test/cmock/vendor/unity
       revision: 031f3bbe45f8adf504ca3d13e6f093869920b091
       remote: throwtheswitch
+    - name: mbedtls
```

Comment:
If this mbedtls is actually administered from the newly added nrfxlib code, maybe this dependency should be placed in a `west.yml` in nrfxlib instead?

Reply:
I think it's better to minimize the number of manifest files. We should ideally just be able to reference the zephyr west.yml from the nrf file, so we don't have to copy / paste the upstream projects we are not forking here, but otherwise having a single manifest file for all the downstream projects will be easier to coordinate in my opinion.

Reply:
Currently, this mbedtls is arm upstream, NOT zephyr version.

So even when Zephyr splits `ext` into projects, this mbebtls is not identical to the Zephyr version.
(Unless Zephyr changes in future)

I agree with @mbolivar we should avoid plcaing `west.yml` in too many locations.

This code:
https://github.com/tejlmand/nrfxlib/blob/2924c24bc82e8d451969080b934c091fb69521f9/nrf_security/CMakeLists.txt#L13

ensures that mbedtls is available as module, while still allowing the user to overwrite with another version.

Reply:
@tejlmand by the way, the multi-manifest design I am working on anticipates this use case. The main (nrf) west.yml will override project attributes (like remote and revision) for any projects which have the same name as ones defined upstream.

Reply:
> The main (nrf) west.yml will override project attributes (like remote and revision) for any projects which have the same name as ones defined upstream.

@mbolivar thinking about that, I wonder what is the best way to handle this, cause in this case, the `mbedtls` pointed to be NCS might not work with regular Zephyr, as Zephyr is actually using its own fork (and including CMake files and in future maybe a `zephyr/module.yml` file also).
And those is not present in arm mbed TLS that we points to.
Something to keep in mind, but not something `west` can solve. It must be the project which somehow determines not just if the module needed is there, but also whether that module suits the need.

Reply:
> And those is not present in arm mbed TLS that we points to.
> Something to keep in mind, but not something west can solve. It must be the project which somehow determines not just if the module needed is there, but also whether that module suits the need.

Worst-case we can make a branch off of upstream with our build system metadata files in it and host it in our GitHub namespace, right?